5.6/.sti/bin/assemble

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+echo "---> Installing application source ..."
+cp -Rf /tmp/src/. ./
+
+if [ -f composer.json ]; then
+  echo "Installing dependencies from composer.json... "
```

Comment:
Bikeshedding... especially as this file is only a copy/paste from the 5.5 one ;)